--- ffmpeg-php-0.6.0/tests/getID3Info.phpt	2008-10-08 20:37:34.000000000 +0300
+++ ffmpeg-php-0.6.0-tests/tests/getID3Info.phpt	2009-06-26 02:23:26.126076603 +0300
@@ -17,6 +17,6 @@
 ffmpeg getTitle(): Test mp3
 ffmpeg getArtist(): Alexander Thomas
 ffmpeg getAlbum(): MP3 test files (http://www.dr-lex.be/software/testsounds.html#Tones)
-ffmpeg getGenre(): (37)
+ffmpeg getGenre(): Sound Clip
 ffmpeg getTrackNumber(): 3
 ffmpeg getYear(): 2008
