--- ffmpeg-php-0.6.0/Makefile	2009-06-26 01:49:03.213173533 +0300
+++ ffmpeg-php-0.6.0/Makefile	2009-06-26 02:22:18.385460514 +0300
@@ -122,7 +122,7 @@
 		done; \
 	fi
 
-PHP_TEST_SETTINGS = -d 'open_basedir=' -d 'output_buffering=0' -d 'memory_limit=-1'
+PHP_TEST_SETTINGS = -d 'open_basedir=' -d 'output_buffering=0' -d 'memory_limit=-1' -d 'safe_mode=0'
 PHP_TEST_SHARED_EXTENSIONS =  ` \
 	if test "x$(PHP_MODULES)" != "x"; then \
 		for i in $(PHP_MODULES)""; do \
@@ -137,10 +137,18 @@
 
 test: all
 	-@if test ! -z "$(PHP_EXECUTABLE)" && test -x "$(PHP_EXECUTABLE)"; then \
+		INI_FILES=`$(PHP_EXECUTABLE) -d 'display_errors=stderr' -r 'echo php_ini_loaded_file(), "\n"; foreach (explode(",\n", trim(php_ini_scanned_files())) as $$a) echo $$a, "\n";'`; \
+		if test "$$INI_FILES"; then \
+			$(EGREP) -hv '^(zend_)?extension(_debug)?(_ts)?[\t\ ]*=' $$INI_FILES > $(top_builddir)/tmp-php.ini; \
+		else \
+			echo > $(top_builddir)/tmp-php.ini; \
+		fi; \
+		ln -snf $(EXTENSION_DIR)/gd.$(SHLIB_DL_SUFFIX_NAME) $(top_builddir)/modules/; \
 		TEST_PHP_EXECUTABLE=$(PHP_EXECUTABLE) \
 		TEST_PHP_SRCDIR=$(top_srcdir) \
 		CC="$(CC)" \
-			$(PHP_EXECUTABLE) $(PHP_TEST_SETTINGS) $(top_srcdir)/run-tests.php -d extension_dir=modules/ $(PHP_TEST_SHARED_EXTENSIONS) tests/; \
+			$(PHP_EXECUTABLE) -d 'display_errors=stderr' $(PHP_TEST_SETTINGS) $(top_srcdir)/run-tests.php -n -w tests.log -d extension_dir=modules/ -d extension=gd.$(SHLIB_DL_SUFFIX_NAME) -d extension=ffmpeg.$(SHLIB_DL_SUFFIX_NAME) -d ffmpeg.allow_persistent=On -c $(top_builddir)/tmp-php.ini tests/; \
+		rm -f $(top_builddir)/modules/gd.$(SHLIB_DL_SUFFIX_NAME); \
 	elif test ! -z "$(SAPI_CLI_PATH)" && test -x "$(SAPI_CLI_PATH)"; then \
 		INI_FILE=`$(top_builddir)/$(SAPI_CLI_PATH) -d 'display_errors=stderr' -r 'echo php_ini_loaded_file();' 2> /dev/null`; \
 		if test "$$INI_FILE"; then \
